Automattic/wp-calypso
# A4A > Marketplace: Update the Pressable page when the refer mode is toggled on

Closes https://github.com/Automattic/jetpack-genesis/issues/458

## Proposed Changes

* This PR updates the Pressable page when the refer mode is on.

## Why are these changes being made?

* To let the users know that the refer mode for Pressable will be added in the future.

## Testing Instructions

* Open the A4A live link
* Go to Marketplace - Hosting - Premier agency hosting > Toggle the refer mode and verify that the below is shown:

<img width="1728" alt="Screenshot 2024-08-07 at 11 24 48 AM" src="https://github.com/user-attachments/assets/c3e13d7f-23a5-4775-bdbd-1cf3154c2cfd">


## Pre-merge Checklist
